canonical/canonical.com
# Rename Operations to Commercial Operations

## Done

- Updated copy as per [doc](https://docs.google.com/document/d/1zLFku18ZAMVStgkIO3bCoKEXW6ShwMEkB0w9d_nCx9U/edit)
- Changed department and slug name from operations to commercial operations
- Added redirect
- Fixed bug wherein multi-word department names were not applying filters correctly and were not showing up on CTAs correctly
- Changed the number for featured roles to a max of 6

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: https://canonical-com-768.demos.haus/careers
- See that the department name is accurate on the departments matrix and click on it
- See that you are routed to the correct page (/careers/commercial-operations)
- Check revised copy against doc
- Click "See all Commercial Operations" roles and see that filters still work as expected
- Visit https://canonical-com-768.demos.haus/careers/operations and https://canonical-com-768.demos.haus/careers/commercial-ops and see that you are redirected to /careers/commercial-operations
- Visit https://canonical-com-768.demos.haus/careers again and click on any department that has a one word name
- Make sure CTA in featured roles works and applies filters on /careers/all as expected 
- Make sure that there are no more than 6 roles listed under "featured roles" for any given department page

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-1139,  https://warthogs.atlassian.net/browse/WD-1198
